extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -47,6 +47,8 @@ import {dev, user} from '../../../../src/log';
 import {createElementWithAttributes} from '../../../../src/dom';
 import {layoutRectLtwh} from '../../../../src/layout-rect';
 import {installDocService} from '../../../../src/service/ampdoc-impl';
+import * as analytics from '../../../../src/analytics';
```

Comment:
Can you explicitly indicate what you require from these imports?  Its the standard behavior.